iree/compiler/Translation/IREEVM.cpp

```diff
@@ -84,6 +85,10 @@ void HighLevelOptimizationOptions::bindOptions(OptionsBinder &binder) {
       llvm::cl::desc(
           "Reduces numeric precision to lower bit depths where possible"),
       llvm::cl::cat(category));
+  binder.opt<bool>("iree-opt-strip-assertions", stripAssertions,
+                   llvm::cl::desc("Strips debug assertions after any useful "
+                                  "information has been extracted."),
+                   llvm::cl::cat(category));
```

Comment:
Want to document this in https://github.com/google/iree/blob/main/docs/website/docs/reference/optimization-options.md / https://google.github.io/iree/reference/optimization-options/? All of the other flags in this file are mentioned there.

@stellaraccident fyi